Symfony/CS/Fixer/Contrib/Php4ConstructorFixer.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+/*
+ * This file is part of the Symfony CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Matteo Beccati <matteo@beccati.com>
+ */
+class Php4ConstructorFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $classes = array_keys($tokens->findGivenKind(T_CLASS));
+
+        for ($i = 0; $i < count($classes); ++$i) {
```

Comment:
It's uncommon in PSR codebase. This tool is designed to fix non-PSR codebase as well, and there it is common.